config/Makefile.linuxgcc

```diff
@@ -9,7 +9,7 @@
 #----------------------------------------------------------------
 
```

Comment:
some change in order to compile it with c++17 , probably need to be done also for other compiler than gcc